bukkit/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '2.0.4'
+}
+
+repositories {
+    maven { url 'https://hub.spigotmc.org/nexus/content/repositories/snapshots/' }
+    maven { url 'https://libraries.minecraft.net/' }
+}
+
+dependencies {
+    compile project(':common')
+
+    compileOnly 'org.spigotmc:spigot-api:1.14-R0.1-SNAPSHOT'
+    compileOnly 'net.kyori:text-adapter-bukkit:3.0.2'
+    compileOnly 'me.lucko:commodore:1.3'
+    compileOnly('net.milkbowl.vault:VaultAPI:1.6') {
+        exclude(module: 'bukkit')
+    }
+    compileOnly 'lilypad.client.connect:api:0.0.1-SNAPSHOT'
```

Comment:
Not needed unless we add direct support for proxies (event driven vs polling for data changes)